esp/src/eclwatch/LZBrowseWidget.js

```diff
@@ -124,6 +124,17 @@ define([
                 context.fileListDialog.hide();
                 context.refreshGrid();
             });
+
+            this.connect(this.uploader, "onError", function (response) {
+                if (response.type === 'error') {
```

Comment:
This is wrapped in event handler that the uploader throws with a type = error which in this case it's the Progress.js by dojox.form.uploader